src/shapes/IText/ITextBehavior.ts

```diff
@@ -404,6 +404,10 @@ export abstract class ITextBehavior<
    * called by {@link canvas#textEditingManager}
    */
   updateSelectionOnMouseMove(e: TPointerEvent) {
+    if (this.__corner) {
+      return;
+    }
```

Comment:
i kind of understand the need o this particular return, but it would be nice if you can add an approximate picture of how your control look like, even a fake one without having to disclose work things, and explain which interaction you make possible that interacts with the mouse move of the selection.